src/Microsoft.Data.Sqlite/project.json

```diff
@@ -21,7 +21,7 @@
     ]
   },
   "dependencies": {
-    "SQLite": "3.12.2",
+    "SQLitePCLRaw.bundle_e_sqlite3": "1.1.0-pre20160928150051",
```

Comment:
There are a couple of options here.

The technique shown in the PR was to take the dependency on a bundle.  This allows me to call SQLitePCL.Batteries_V2.Init() from Microsoft.Data.Sqlite itself (in a static constructor) in portable code.  The bait-and-switch technique replaces this with a real implementation.  This is accomplished by adding the bundle to the tests assembly.

The main reason I used the technique above was that it made the xUnit tests work without sprinkling Init() calls into them.

Another option would be to make Microsoft.Data.Sqlite depend on SQLitePCLRaw.core (the main assembly) and to do the Init() call somewhere else, more explicitly.